tests/terraform/runner/test_plan_runner.py

```diff
@@ -170,7 +170,7 @@ def test_runner_root_dir(self):
         self.assertEqual(report.get_exit_code(soft_fail=False), 1)
         self.assertEqual(report.get_exit_code(soft_fail=True), 0)
 
-        self.assertGreaterEqual(report.get_summary()["failed"], 85)
+        self.assertGreaterEqual(report.get_summary()["failed"], 82)
```

Comment:
how did this number change?

Reply:
Signs of bad stuff happening... branch is not working anyhow